packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -230,6 +234,9 @@ class SelectableRegion extends StatefulWidget {
   /// [TextSelectionControls] implementation with no controls.
   final TextSelectionControls selectionControls;
 
+  /// Called when the selected content changes.
+  final SelectionChangeEventListener? onSelectionChange;
```

Comment:
I just realize you can use ValueChanged<SelectedContent?>, please use that one instead, sorry for the confusing